### PR TITLE
Add SEA Equipment and Gun racks

### PR DESCRIPTION
**Pull request by KalimbaMachine**
## About the PR

Adds the SEA vendors with minor additions that deviate from parity: 
 *Adds MP Security Glasses

Removed Alpha Headset from SEATech Vendor
(Removed the Officer's Jacket)

## Media

Updated pictures showing limited quantity:
seaequip11
seaequip12
seaweapon11

## Requirements

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added SEA Equipment and Gun Racks!

### CODE REVIEW

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
These are the wrong types of vendors, this pr will not be merged half finished, but thank you for doing the fills.

Also please remove the officer jacket as it's not part of their uniform for the sake of recognizability.

Sec glasses can stay.

**Comment by KalimbaMachine** [comment]
> These are the wrong types of vendors, this pr will not be merged half finished, but thank you for doing the fills.
> 
> Also please remove the officer jacket as it's not part of their uniform for the sake of recognizability.
> 
> Sec glasses can stay.

do you want the kind that have limited selection? or what kind should i be using here?
How can i improve it.

**Comment by QuietlyWhisper** [comment]
> do you want the kind that have limited selection? or what kind should i be using here? How can i improve it.

It should be the limited selection yes, all SEA players can access these vendors and get equipment from them, they work the same as the rifleman armor vendor when done correctly.
see `Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/rifleman.yml` for how to implement it correctly

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/senior_enlisted_advisor.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  - type: CMAutomatedVendor
+    sections:
+    - name: Personal Primary
+      choices: { CMPrimary: 1 }
+      entries:
+      - id: WeaponRifleM54C
+    - name: Ammunition
+      entries:
+      - id: CMMagazineRifleM54C
+        amount: 10
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/senior_enlisted_advisor.yml:99]: This costs points, it's not a fixed amount
```suggestion
        points: 9
```

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**